sdk/spring/spring-cloud-azure-feature-management/src/main/java/com/azure/spring/cloud/feature/management/implementation/FeatureManagementProperties.java

```diff
@@ -105,13 +123,37 @@ private void addToFeatures(Map<? extends String, ? extends Object> features, Str
         }
     }
 
+    private void addServerSideFeature(Map<? extends String, ? extends Object> features, String key) {
+        final Object featureValue = features.get(key);
+
+        ServerSideFeature serverSideFeature = null;
+        try {
+            serverSideFeature = MAPPER.convertValue(featureValue, ServerSideFeature.class);
+        } catch (IllegalArgumentException e) {
+            LOGGER.error("Found invalid feature {} with value {}.", key, featureValue.toString());
+        }
+
+        if (serverSideFeature != null && serverSideFeature.getId() != null) {
+            if (serverSideFeature.getConditions() != null && serverSideFeature.getConditions().getClientFilters() != null
```

Comment:
`serverSideFeature.getConditions().getClientFilters().size() > 0` is allowed to be 0. This should result in a feature flag returning false when enabled, as no feature filter returned true.

This is how the current library works.

Reply:
Updated.